packages/patternfly-4/react-core/src/components/Select/CheckboxSelectGroup.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styles from '@patternfly/patternfly/components/Select/select.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** Checkboxes within group */
+  children: PropTypes.node,
+  /** Additional classes added to the CheckboxSelectGroup control */
+  className: PropTypes.string,
+  /** Group label */
+  label: PropTypes.string,
+  /** Internal ID for group title */
+  titleId: PropTypes.string,
```

Comment:
this says it is internal.  If so, we should not be exposing it to the consumer.

Reply:
I can remove it. I think the linter just complains about missing variables in the PropTypes declaration without it however, and I have gotten feedback before to add these kind of variables before. It would be nice to have a way to leave out these internal props when it gets translated into our docs.